src/libpriv/rpmostree-refts.cxx

```diff
@@ -114,6 +123,29 @@ RpmTs::package_meta (const rust::Str name) const
           retval->_size = headerGetNumber (h, RPMTAG_LONGARCHIVESIZE);
           retval->_buildtime = headerGetNumber (h, RPMTAG_BUILDTIME);
           retval->_src_pkg = headerGetString (h, RPMTAG_SOURCERPM);
+
+          struct rpmtd_s nchanges_date_s;
+          _cleanup_rpmtddata_ rpmtd nchanges_date = NULL;
+          nchanges_date = &nchanges_date_s;
+          headerGet (h, RPMTAG_CHANGELOGTIME, nchanges_date, HEADERGET_MINMEM);
+          int ncnum = rpmtdCount (nchanges_date);
+          if (!ncnum)
+            ncnum = 0;
+          int frequency_recent_updates = 0;
+          GDateTime *now_dt = g_date_time_new_now_utc ();
+          while (ncnum > 0)
+            {
+              uint64_t nchange_date = 0;
+              rpmtdNext (nchanges_date);
+              nchange_date = rpmtdGetNumber (nchanges_date);
+              GDateTime *build_dt = g_date_time_new_from_unix_utc (nchange_date);
+              if (g_date_time_compare (build_dt, g_date_time_add_years (now_dt, -1)))
```

Comment:
This logic is ignoring updates older than a year, right?  That seems very deserving of a comment describing that and why we do it.